test/lightning_web/live/job_live_test.exs

```diff
@@ -68,4 +70,60 @@ defmodule LightningWeb.JobLiveTest do
                "@other_org/some_module"
     end
   end
+
+  describe "Deleting a job from inspector" do
+    test "jobs with no dowstream jobs can be deleted", %{
+      conn: conn,
+      project: project,
+      job: job
+    } do
+      {:ok, view, html} =
+        live(
+          conn,
+          Routes.project_workflow_path(conn, :edit_job, project.id, job.id)
+        )
+
+      assert html =~ project.name
+
+      assert has_element?(view, "#delete-job")
+
+      view
+      |> element("#delete-job")
+      |> render_click()
+
+      assert_patched(
+        view,
+        Routes.project_workflow_path(conn, :show, project.id)
+      )
+    end
+
+    test "jobs with downstream jobs can't be deleted", %{
+      conn: conn,
+      project: project,
+      job: job
+    } do
+      {:ok, %Job{} = _} =
+        Jobs.create_job(%{
+          body: "some body",
+          enabled: true,
+          name: "some name",
+          adaptor: "@openfn/language-common",
+          trigger: %{type: "on_job_success", upstream_job_id: job.id},
+          workflow_id: job.workflow_id
```

Comment:
What about the functions from `JobFixtures`? Can you try replace the call to `create_job` with `workflow_job_fixture` from the fixtures?